slash/css/package.json

```
@@ -7,6 +7,7 @@
   "exports": {
     "./logo-axa.svg": "./files/assets/logo-axa.svg",
     "./dist/common/assets/logo-axa.svg": "./files/assets/logo-axa.svg",
+    "./spinner.svg": "./files/assets/spinner.svg",
```

il me semble que 
```suggestion
"./*.svg": "./files/assets/*.svg",
```
devrait marché